pyramid_oereb/standard/pyramid_oereb.yml.mako

```diff
@@ -131,7 +131,13 @@ pyramid_oereb:
     canton: BL
     # Mapping for other optional attributes
     mapping:
-      municipality: subtype
+      official_number: number
+      abbreviation: abbreviation
```

Comment:
Only for my understanding, why do we need to map the `abbreviation` (and not the other) (For instnce, the title is automatically mapped, no ?)

Reply:
All mandatory attributes are mapped automatically. For the optional attributes a mapping has to be defined, because their meaning might be different for each OEREBlex installation.

Reply:
Ok, thanks :-)